common.gypi

```diff
@@ -89,7 +89,7 @@
         ],
         'msvs_settings': {
           'VCCLCompilerTool': {
-            'RuntimeLibrary': 0, # static release
```

Comment:
@piscisaureus The Mapbox Studio app (https://www.mapbox.com/mapbox-studio/) uses Node. And it depends on node c++ modules that need to be linked to the dll version (and not statically linked to the c++ runtime) like node-mapnik. So, I build node against the same dll to ensure all C++ code is using the same exact one. Anyway, I realize this is not what node core is going to do and I did not mean for this pull to be against joyent/node as per my comment at https://github.com/joyent/node/pull/25780#issuecomment-126083384. Sorry for the noise.

BTW, this pull does have some fixes needed to compile against Visual 2015 which I will - at some point when I have time - extract and provide as fixes separately.